os-eap-launch/added/launch/maven-repos.sh

```diff
@@ -177,3 +182,12 @@ function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
 
+function set_local_repo_path() {
+    local settings="${1}"
+    local local_path="${2}"
+    local xml="\n\
+    <localRepository>${local_path}</localRepository>\n\
+    <!-- ### configured local repository ### -->"
```

Comment:
We don't need to put the "configured local repository" comment line back in here, because multiple search and replaces are not supported. So I think Line 187 through 190 could just be replaced with:
local xml="<localRepository>${local_path}</localRepository>"